src/pages/ResearcherProfile.js

```diff
@@ -254,8 +269,8 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
   }
 
   isValidState(value) {
-    const stateSelected = (!isNil(value) || !isEmpty(value)) && (value !== "N/A");
-    const inUS = (this.state.profile.country === "United States of America");
+    const stateSelected = (!isNil(value) || !isEmpty(value));
```

Comment:
This is reporting valid even if I unselect the state option so validation is failing here.